src/log.rs

```diff
@@ -63,6 +67,65 @@ pub fn log<W, F>(rq: &Request, mut output: W, f: F) -> Response
     }
 }
 
+
+/// Calls custom logging functions after processing a request.
+///
+/// This is nearly identical to the [`rouille::log`](fn.log.html) function except it
+/// takes two logging functions that will be called with access to the request/response
+/// structs and the total execution duration of the handler.
+///
+/// # Example
+///
+/// ```
+/// #[macro_use] extern crate log;
+/// extern crate chrono;
+/// # extern crate rouille;
+/// use rouille::{Request, Response};
+///
+/// fn log_ok(req: &Request, resp: &Response, _elap: std::time::Duration) {
+///     info!("{} {}", req.method(), req.raw_url());
+/// }
+///
+///
+/// fn handle(request: &Request) -> Response {
+///     let now = chrono::Utc::now().format("%Y-%m-%d %H:%M:%S%.6f");
+///     let log_err = |req: &Request, _elap: std::time::Duration| {
```

Comment:
Should be next to `log_ok`.

Reply:
I've moved `log_ok` down next to `log_err`. Wanted to make a point that you need to capture the current time to get the proper timestamp of the request since you can't be sure when the log functions will run (unless you do datetime math in the log function).